cargo-pgx/src/command/schema.rs

```diff
@@ -502,6 +502,14 @@ fn create_stub(
 
     let mut so_rustc_invocation = Command::new("rustc");
     so_rustc_invocation.stderr(Stdio::inherit());
+
+    if let Some(rustc_flags_str) = std::env::var("RUSTFLAGS").ok() {
```

Comment:
(I note for @workingjubilee this is unrelated to this PR itself, it appears to be from 0.4.2 release.)